### DISCUSSION TITLE
Anyone is using Pavlovia? Problem with French accent display...

### DISCUSSION THREAD

**Discussion by inesmtc** [jspsych/jsPsych#765] [Q&A] [answered]
Hi everyone,

I have some issue using Pavlovia to publish my experiment. It's in French so the instructions contain some French accents such as é, è, à,... Everything seemed to work well using Chrome or Firefox (not with Edge but that's not a problem). The texts are correctly displayed when I run the experiment locally. 
I put it on the Pavlovia servers and now the accents aren't recognized anymore. I write this here because it seems that several projects that use PsychoJS don't have the same problem with Pavlovia... I guess that the problem comes from de jsPsych-Pavlovia compatibility... But I really don't know.

Thanks by advance!
I.M.

**Comment by kupiqu** [comment]
I encountered what I think it's the same issue with the ndash symbol `–`, I ended up putting the symbol in html code. This can be a workaround but it would be great if we wouldn't have to.

**Comment by vijaymarupudi** [comment, marked as answer]
Can you make sure that you have the meta charset tag in the html head `<meta charset='utf-8'>`

**Reply by inesmtc** [reply]
Wouah thank you so much! It seems to work now

**Reply by kupiqu** [reply]
Yep, it worked, thank you @vijaymarupudi
